utils/file_manager.py

```
@@ -0,0 +1,131 @@
+    "claude": "claude_desktop_config.json",
+    "cursor": "mcp.json"
+}
+
+class ConfigFinder():
+    def __init__(self, agentName):
+        self.agentName = AGENT_CONFIG_FILENAMES[agentName]
+
+    def get_path(self):
+        """Searches normal locations for the config file"""
+        potential_paths = []
+        config_filename = self.fileName
```

Pylint (3.3.7)

[error] Instance of 'ConfigFinder' has no 'fileName' member

(E1101)